src/fragments/start/getting-started/reactnative/auth.mdx

```diff
@@ -0,0 +1,215 @@
+
+Amplify uses [Amazon Cognito](https://aws.amazon.com/cognito/) as the main authentication provider. Amazon Cognito is a robust user directory service that handles user registration, authentication, account recovery & other operations. In this tutorial, you'll learn how to add authentication to your application using Amazon Cognito and username/password login.
+
+## Create authentication service
+
+To add authentication to your app, run this command:
+
+```bash
+amplify add auth
+```
+
+Select the defaults for the following prompts:
+
+```console
+? Do you want to use the default authentication and security configuration? Default configuration
+Warning: you will not be able to edit these selections. 
+? How do you want users to be able to sign in? Username
+? Do you want to configure advanced settings? No, I am done.
+```
+
+To deploy the service, run the `push` command:
+
+```bash
+amplify push
+```
+
+Now, the authentication service has been deployed and you can start using it. To view the deployed services in your project at any time, go to Amplify Console by running the following command:
+
+```bash
+amplify console
+```
+
+## Create login UI
+
+Creating a login flow can be quite difficult and time consuming to get right. Luckily, Amplify UI has an [authentication](https://ui.docs.amplify.aws/react-native/connected-components/authenticator) component that provides an entire authentication flow for you, using the configuration you specified in **aws-exports.js.**
+
+### Install Amplify UI
+
+The `@aws-amplify/ui-react-native` package includes React Native specific UI components you'll use to build your app. Install it with this:
+
+```bash
+npm i @aws-amplify/ui-react-native aws-amplify react-native-safe-area-context amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill
+```
+
+### Add the Amplify UI Authenticator component
+
+<BlockSwitcher>
+<Block name="Expo CLI">
+
+Open **App.js** and make the following changes:
+
+</Block>
+<Block name="React Native CLI">
+
+Open **App.ts** and make the following changes:
+
+</Block>
+</BlockSwitcher>
+
+1. Import the `withAuthenticator` component and `useAuthenticator` hook:
+
+```javascript
+
+import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react-native';
+
+````
+
+2. Pass `{signOut, user}` to the any component where you need it (e.g. `SignOutButton`):
+
+```javascript
+
+const SignOutButton = () => {
+  const { user, signOut } = useAuthenticator((context) => [context.user]);
+  return (
+    <Pressable onPress={signOut} style={styles.buttonContainer}>
+      <Text style={styles.buttonText}>Hello, {user.username}! Click here to sign out!</Text>
+    </Pressable>
+  )
+};
+
+```
+
+3. Add `SignOutButton` component to  your `App` component:
+
+```javascript
+
+// ...
+return (
+    <View style={styles.container}>
+      <StatusBar hidden/>
+      <SignOutButton />
```

Comment:
```suggestion
  <View style={styles.container}>
    <SignOutButton />
```

Fixed spacing and also recommend removing the StatusBar component as it is not imported into the project and not needed